### ISSUE TITLE
[BUG] No back-channel audio from web test page to master

### ISSUE THREAD

**Issue by jvanvorst** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#1583]
**Logging**
Add relevent SDK logging.  IMPORTANT NOTE:  Please make sure to NOT share AWS access credentials under any circumstance!  Please make sure they are not in the logs.

**Describe the bug**
kvsWebrtcClientMaster does not seem to be receive back channel audio from web test page.

**SDK version number**
master branch

**To Reproduce**
1. export AWS credentials
2. export AWS_KVS_LOG_LEVEL=1
3. run kvsWebrtcClientMaster <channel>
4. on web test page at https://awslabs.github.io/amazon-kinesis-video-streams-webrtc-sdk-js/examples/index.html,  select 'send audio'
5. click 'Start Viewer' to start client
6. click 'allow' when prompted for mic access
7. Audio and video is sent from master to web test page
8. No audio is received on master

**Expected behavior**
Audio frames should be received on the master side.
The only back channel data received is when 'send video' is selected on the web test page, and logs are of the form:
2022-11-09 00:05:08 VERBOSE sampleFrameHandler(): Frame received. TrackId: 547744464128, Size: 1167, Flags 2030010844
2022-11-09 00:05:08 VERBOSE sampleFrameHandler(): Frame received. TrackId: 547744435520, Size: 1269, Flags 2030010844
2022-11-09 00:05:09 VERBOSE sampleFrameHandler(): Frame received. TrackId: 547744480128, Size: 1290, Flags 2030010844
2022-11-09 00:05:09 VERBOSE sampleFrameHandler(): Frame received. TrackId: 547744451520, Size: 1835, Flags 2030010844
2022-11-09 00:05:09 VERBOSE sampleFrameHandler(): Frame received. TrackId: 547744520800, Size: 1013, Flags 2030010844

**Comment by disa6302** [comment]
@jvanvorst , 

Do you see any errors in the logs? Can you run `export DEBUG_LOG_SDP=TRUE` to log the SDPs as well?

**Comment by jvanvorst** [comment]
[kvsFileLog.0.txt](https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/files/10059518/kvsFileLog.0.txt)

Attached is the log after
export DEBUG_LOG_SDP=TRUE
export AWS_ENABLE_FILE_LOGGING=TRUE
export AWS_KVS_LOG_LEVEL=2
on master side

Start master
Select 'Send Audio' in the test page
Click 'Start Viewer'

**Comment by bneff** [comment]
Hi KVS Team, I am have the same issue as @jvanvorst.

This is a snippet of the SDP from the Master ( KVS ) with some items redacted.  Note that is does state sendrecv.
In Chrome, I also see the the outbound-rtp active and sending packets. 

m=audio 9 UDP/TLS/RTP/SAVPF 0
c=IN IP4 127.0.0.1
a=candidate:0 1 udp XXX
a=msid:myKvsVideoStream myAudioTrack
a=ssrc:1780384562 cname:mF2EIEY0g9MhFzyh
a=ssrc:1780384562 msid:myKvsVideoStream myAudioTrack
a=ssrc:1780384562 mslabel:myKvsVideoStream
a=ssrc:1780384562 label:myAudioTrack
a=rtcp:9 IN IP4 0.0.0.0
a=ice-ufrag:U2CJ
a=ice-pwd:XXX
a=ice-options:trickle
a=fingerprint:sha-256 XXX
a=setup:active
a=mid:0
a=sendrecv
a=rtcp-mux
a=rtcp-rsize
a=rtpmap:0 PCMU/8000
a=rtcp-fb:0 nack
a=rtcp-fb:0 goog-remb
a=rtcp-fb:0 transport-cc

**Comment by jvanvorst** [comment]
This also happens running two instances of the test page, one as master on machine 1, and one as viewer on machine 2.  Audio and video go from master to viewer, but there is no audio from viewer back to master.

**Comment by bneff** [comment]
I'm not sure what changed on my side, but I'm now able to receive audio sent by the KVS WebRTC Test Page running Chrome Version 108.0.5359.94 (Official Build) (arm64).  The receiving app is the kvsWebRTCClientMaster.c that has been modified for my platform.

To assist @jvanvorst , I should mention that I added a section in the sampleReceiveVideoFrame to register an audioFrame callback.  By default, sampleReceiveVideoFrame only registers a handler for the video track.

something like:

retStatus = transceiverOnFrame(pSampleStreamingSession->pAudioRtcRtpTransceiver, (UINT64) pSampleStreamingSession, audioFrameHandler);

Then of course define the audioFrameHandler.  I added mine to Common.c.
